OGGExtractor.cpp

```diff
@@ -296,11 +307,11 @@ void OGGExtractor::extractFiles()
       name = name.replace(QRegExp("[^a-zA-Z0-9_- ]"),QString(""));
       if(name.isEmpty())
       {
-        name = tr("found_ogg_%1").arg(i+1);
+        name = getDefaultOutputFilename(i+1,data);
       }
 
       QDir dir(destination);
-      QFile file(dir.absoluteFilePath(tr("%1 - %2.ogg").arg(i + 1, numberLength, 10, QChar('0')).arg(name)));
+      QFile file(dir.absoluteFilePath(tr("%1.ogg").arg(name)));
```

Comment:
If the user doesn't change the name of the file there is no problem (as the new 'default' is unique), but if the user renames it could overwrite an existing file with the same name, the variable **i** is precisely to ensure an unique output filename (between the detected ogg files).